modules/core/nuxeo-core-el/src/main/java/org/nuxeo/ecm/platform/el/ExpressionContext.java

```diff
@@ -30,6 +32,15 @@
 
 public class ExpressionContext extends ELContext {
 
+    private static final Map<String, Method> DEFAULT_FUNCTIONS = new HashMap<>();
+
+    static {
```

Comment:
You could use `Collectors.toUnmodifiableMap` to directly define the value of `DEFAULT_FUNCTIONS` without a static block.